src/main/java/io/supertokens/utils/Utils.java

```diff
@@ -444,4 +445,21 @@ public static boolean containsUrl(String urlToCheckIfContains, String whatItCont
 
         return originalHost.equals(wantedHost);
     }
+
+    public static String camelCaseToSnakeCase(String toSnakeCase) {
+        String regex = "([a-z])([A-Z]+)";
+        String replacement = "$1_$2";
+        toSnakeCase = toSnakeCase
+                .replaceAll(
+                        regex, replacement)
+                .toLowerCase();
+        return toSnakeCase;
+    }
+
+    public static String snakeCaseToCamelCase(String toCamelCase) {
+        toCamelCase = Pattern.compile("_([a-z])")
+                .matcher(toCamelCase)
+                .replaceAll(m -> m.group(1).toUpperCase());
+        return toCamelCase;
+    }
```

Comment:
need tests for these

Reply:
Sure, I agree!

Reply:
I dont see any tests added for this, so keeping this open